### PR TITLE
README: update deploy docs

### PR DESCRIPTION
**Pull request by david-caro**
Bug: T331330

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,21 +47,40 @@ or want to examine things more, use `go test -test.v ./...`
...
+> cookbook wmcs.toolforge.k8s.component.build --git-url https://github.com/toolforge/buildpack-admission-controller
 
-    `myuser@tools-k8s-control-1:# ./deploy.sh <tools or toolsbeta>`
 
-  to deploy to toolforge or toolsbeta.
+That will give you an image tag:
+```
+dcaro@vulcanus$ cookbook wmcs.toolforge.k8s.component.build --git-url https://github.com/toolforge/buildpack-admission-controller
+...
+[DOLOGMSG]: build & push docker image docker-registry.tools.wmflabs.org/toolforge-buildpack-admission-controller:f90bd8f from https://github.com/toolforge/buildpack-admission-controller (f90bd8f)
```
Comment (supertassu) [inline README.md:68]: side note, the image should be tagged just `:buildpack-admission` to match the previous versions. the cookbook has a list of overrides, buildpack-admission just needs to be added here

**Review by supertassu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (david-caro) [reply README.md:68]: Do you mean using the `--docker-image-tag`  of the cookbook?
Can you elaborate on why it should be specially tagged (isn't that creating the same issue as the `latest` tag?)

Comment (supertassu) [reply README.md:68]: Sorry, meant the image name, not the tag.

Comment (david-caro) [reply README.md:68]: Hmm, I'm thinking that maybe we want to do the other way around, use the buildpack-admission-controller image name everywhere as it's the default name generated (less options to remember to use when running)

**Review by blancadesal — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (david-caro) [reply README.md:68]: Done :), I'll clean up the old image after deploying :+1:

**Pull request merged**